### ISSUE TITLE
[v4] iOS page transitions don't work properly with multiple ion-toolbars

### ISSUE THREAD

**Issue by sbannigan** [ionic-team/ionic#16262]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
Ionic:

   ionic (Ionic CLI)          : 4.1.1
   Ionic Framework            : @ionic/angular 4.0.0-beta.15
   @angular-devkit/core       : 0.7.5
   @angular-devkit/schematics : 0.7.2
   @angular/cli               : 6.1.2
   @ionic/ng-toolkit          : not installed
   @ionic/schematics-angular  : not installed

System:

   NodeJS : v8.11.2
   npm    : 6.4.1
   OS     : macOS
```

**Describe the Bug**
When using multiple toolbars, the page change animation on iOS doesn't work as expected. Attached a gif of this bug, with the animation slowed down. The bottom toolbar just appears during the enter animation and then overlaps the previous screen and then disappears during the leave animation.

![2018-11-07 16 57 55](https://user-images.githubusercontent.com/15218748/48164774-7a13b200-e2b1-11e8-984b-97901e8106bf.gif)

**Steps to Reproduce**
1. Navigate from a page with only 1 toolbar to a page with more than 1.
2. Navigate back to the page with 1 toolbar.

**Expected Behavior**
All ion-toolbars inside ion-header should animate smoothly forward and backward including any type of component inside the toolbar like the ion-segment

**Comment by kitolog** [comment]
I have the similar behaviour. My case is show / hide header on different pages.

**Comment by elmartino** [comment]
+1

**Comment by zakton5** [comment]
+1

**Comment by marjan-wehelpen** [comment]
bug is still there on 4.0.0-rc.0

**Comment by kianliang-iium** [comment]
+1

**Comment by jayordway** [comment]
+1

**Comment by zakton5** [comment]
I put in a PR for this so hopefully it gets merged in soon.

**Comment by liamdebeasi** [comment]
Hi everyone,

This issue has been resolved via #17224 and will be in the next release of Ionic. We pushed a nightly build of Ionic with this fixed (`npm i @ionic/core@dev @ionic/angular@dev`). Any feedback on these changes would be greatly appreciated!

Thanks!

**Comment by liamdebeasi** [comment]
Hi everyone,

A fix for this has been shipped with the release of Ionic 4.1.2. You can get the latest by running `npm i @ionic/core@latest @ionic/angular@latest`. 

If you encounter any new issues, feel free to open a new issue.

Thanks!
